modules/persistent-storage-csi-snapshots-restore.adoc

```diff
@@ -30,7 +30,7 @@ spec:
   dataSource:
     name: mysnap <1>
     kind: VolumeSnapshot <2>
-    apiGroup: snapshot.storage.k8s.io/v1 <3>
+    apiGroup: snapshot.storage.k8s.io <3>
```

Comment:
Please also remove `/v1` from the annotated note on line 42. Other than that, this LGTM.